dot/rpc/modules/author_test.go

```diff
@@ -298,36 +346,55 @@ func TestAuthorModule_PendingExtrinsics(t *testing.T) {
 				logger:     log.New(log.SetWriter(io.Discard)),
 				txStateAPI: mockTransactionStateAPI,
 			},
-			args: args{
-				res: new(PendingExtrinsicsResponse),
-			},
 			wantRes: PendingExtrinsicsResponse{
 				common.BytesToHex([]byte("someExtrinsic")),
 				common.BytesToHex([]byte("someExtrinsic1")),
 			},
 		},
 	}
 	for _, tt := range tests {
+		res := PendingExtrinsicsResponse{}
+		tt.args.res = &res
```

Comment:
Can you put these in the subtest please? It looks strange to be above here, and might not work with future eventual parallel subtests